templates/source/view.html

```diff
@@ -6,28 +6,29 @@
         <div class="jumbotron">
             <div class="row">
                 <div class="col-sm-12">
-                    <h2 class="page-title cited">
-                        <a href="{{ source.source_url }}" target="_blank">
-                            {{ source.title }}
-                        </a>
-                        {% if request.user.is_staff %}
+                    {% if request.user.is_staff %}
+                        <h2>
                             <small class="pull-right">
                                 <a class="btn btn-default" type="button" role="button" href="{% url 'update-source' source.uuid %}">
                                     <i class="fa fa-fw fa-edit"></i>{% trans "Edit" %}
                                 </a>
                             </small>
-                        {% endif %}
+                        </h2>
+                    {% endif %}
+                    <h2 class="page-title cited">
+                        <i class="fa fa-fw fa-paperclip"></i>
+                        <a href="{{ source.source_url }}" target="_blank">
+                            {{ source.title }}
+                        </a>
                     </h2>
                 </div>
             </div>
             <div class="row">
                 <div class="col-sm-12">
                     <p>
-                        <strong>Publisher: </strong>{{ source.publication }} ({{ source.publication_country }})<br />
-                        <strong>Publication date: </strong>{{ source.published_on }}<br />
-                        <strong>Archive URLs: </strong>{{ source.archive_urls }}<br />
-                        <strong>Accessed on: </strong>{{ source.accessed_on }}<br />
```

Comment:
`Archive URLs` and `Accessed on` are attributes of access points, not sources. Since this PR improves the display name of access points to provide this information, we no longer need to present it as top-level source metadata.